src/components/Prompt.tsx

```diff
@@ -92,6 +109,13 @@ export function Prompt({
               crossAxisAlignment="flex-start"
               borderRadius="6px"
               borderLeftWidth={isLast ? "4px" : "0px"}
+              position="relative"
+              onMouseEnter={() =>
+                !(data.fluxNodeType === FluxNodeType.User) && setHoveredNodeId(node.id)
+              }
+              onMouseLeave={() =>
+                !(data.fluxNodeType === FluxNodeType.User) && setHoveredNodeId(null)
+              }
```

Comment:
Two logical operators instead of one?
```suggestion
              onMouseEnter={() =>
                data.fluxNodeType !== FluxNodeType.User && setHoveredNodeId(node.id)
              }
              onMouseLeave={() =>
                data.fluxNodeType !== FluxNodeType.User && setHoveredNodeId(null)
              }
```

Reply:
Good catch. Fixed this and moved copyToClipboard to a util file